lib/events.js

```diff
@@ -52,11 +52,13 @@ Object.defineProperty(EventEmitter, 'defaultMaxListeners', {
   get: function() {
     return defaultMaxListeners;
   },
-  set: function(arg) {
+  set: function(n) {
     // force global console to be compiled.
     // see https://github.com/nodejs/node/issues/4467
     console;
-    defaultMaxListeners = arg;
+    if (typeof n !== 'number' || n < 0 || isNaN(n))
```

Comment:
Could we replace `isNaN(n)` with `n !== n`, possibly with a comment with a short explanation?